src/kpatch_user.c

```diff
@@ -627,11 +625,41 @@ server_stress_test(int fd, int argc, char *argv[])
 	return 0;
 }
 
+static int stress_test_log_init(int pid)
+{
+	if (!strlen(test_log_base))
+		return 0;
+	if (pid < 0) {
+		if (snprintf(test_log_name, LOG_NAME_LEN, "%s-base", test_log_base) >= LOG_NAME_LEN) {
+			kperr("Can't initialize log \'%s\'-<PID>\n", test_log_base);
+			return -1;
+		}
+	} else
+		if (snprintf(test_log_name, LOG_NAME_LEN, "%s-%d", test_log_base, pid) >= LOG_NAME_LEN) {
+			kperr("Can't initialize log \'%s\'-<PID>\n", test_log_base);
+			return -1;
+		}
+	if (log_file_init(test_log_name)) {
+		kperr("Can't open log file \'%s\'\n", test_log_name);
+		return -1;
+	}
+	return 0;
+}
+
 static int cmd_stress_test(int fd, int argc, char *argv[])
 {
+	int pid;
+	if (sscanf(argv[1], "%d", &pid) != 1) {
+		kperr("Can't parse pid from %s\n", argv[1]);
+		return -1;
+	} else
+		kpinfo("Fork for pid=%d\n", pid);
+
 	int child = fork();
 	if (child == 0) {
-		int rv = server_stress_test(fd, argc, argv);
+		stress_test_log_init(pid);
```

Comment:
This returns value. Check it.

Reply:
Error messages are written inside stress_test_log_init.
Contrary to base process, children should continue work even if can't initialize log file. 
Is it better for them to exit in case of logging error too?